std_msgs/Header header
bb_pub_node/BoundingBox2D[] boxes
